app/templates/components/modals/change-image-modal.hbs

```diff
@@ -0,0 +1,49 @@
+<i class="black close icon"></i>
+<div class="header">
+  {{t 'Update image'}}
+</div>
+<br>
+<br>
+<div class="ui grid">
+  <div class="row">
+    <div class="sixteen wide column center aligned">
+      <div class="ui labeled input">
+        <div class="ui label">
+          {{t 'Placeholder'}}
+        </div>
+        <input type="file" name="updateImage">
+      </div>
+      <br>
+      <br>
+      <div class="ui labeled input">
+        <div class="ui label">
+          {{t 'Copyright Information'}}
+        </div>
+        <input type="text" name="copyright">
+      </div>
+      <br>
+      <br>
+      <div class="ui labeled input">
+        <div class="ui label">
+          {{t 'Origin Information'}}
+        </div>
+        <input type="origin" name="origin">
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="sixteen wide column center aligned">
+      <p>
+        For Cover Photos : We recommend using at least a 1300x550px (2:1 ratio) image.
+        For Avatar Photos : We recommend using at least a 360x360px (1:1 ratio) image.
+        The images should be no larger than 10MB.
+      </p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="six wide column right floated right aligned" style="margin-bottom: 2%">
```

Comment:
Use `hidden divider` instead of inline style. Refer to https://semantic-ui.com/elements/divider.html. You can add it below this div for extra space.